Tools/Scripts/webkitpy/common/net/bugzilla/results_fetcher.py

```diff
@@ -207,3 +208,28 @@ def lookup_ews_results_from_bugzilla(
         raise RuntimeError(msg)
 
     return lookup_ews_results_from_bugzilla_patch(patch, bot_filter_name)
+
+
+def lookup_ews_results_from_pr(pr):
+    urls = [status.url for status in pr.statuses]
+    return lookup_ews_results(urls)
```

Comment:
Given we might have other status checks, do we want to somehow filter this down to things from EWS? Hardcoding the URL seems bad, but it's unclear what else we can do here.